vip-cache-manager/vip-cache-manager.php

```diff
@@ -239,6 +242,18 @@ public function execute_purges() {
 		 */
 		do_action( 'wpcom_vip_cache_pre_execute_bans', $this->ban_urls );
 
+		$num_ban_urls = count( $this->ban_urls );
+		$num_purge_urls = count( $this->purge_urls );
+		if ( $num_ban_urls >= self::MAX_BAN_URLS ) {
```

Comment:
I think there's an off-by-one error here. The condition should be `>` instead of `>=`. As is, this would generate an error if 10 URLs are provided, but all 10 would still be processed?

Reply:
Good catch!